FEWD/Week03/lesson05/homework/week3-relaxr-part2/css/style.css

```diff
@@ -0,0 +1,211 @@
+/*
+Project Name: Blank Template
+Client: Your Client
+Author: Your Name
+Developer @GA in NYC
+*/
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+ box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:before, .clearfix:after {
+  content: "";
+  display: table;
+}
+
+.clearfix:after{
+  clear: both;
+}
+
+.alignright {
+  float: right;
+  padding: 0 0 10px 10px; /* note the padding around a right floated image */
+}
+
+.alignleft {
+  float: left;
+  padding: 0 10px 10px 0; /* note the padding around a left floated image */
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+body {
+  background-color: #f0efef;
+  color: #000;
+  font-size: 12px;
+  line-height: 1.4;
+  font-family: 'Open Sans', Helvetica, Arial, sans-serif;
+  width: 100%;
+}
+
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+header {
+  background-color: #033048;
+  height: 80px;
+}
+
+h1 {
+  font-size: 40px;
+  color: white;
+  font-weight: 700;
+  padding: 10px 0 0 70px;
+  float: left;
+}
+
+nav {
+  float: right;
+  padding-right: 75px;
+}
+
+nav li {
+  font-size: 18px;
+  display: inline-block;
+  padding-right: 30px;
+  padding-top: 28px;
```

Comment:
So, I can see a few lines above (on line 58) that you limited the height of the `<header>` to 80px. There's nothing really wrong with that, especially if you're trying to vertically align the children using `vertical-align: middle;`. However `vertical-align: middle;` doesn't work on items that are floated, like the two children of your header (`<h1>` and `<nav>`) are here.

So instead, you might want to consider letting the `<header>` simply be as tall as its children. Which leads me to the comment I'm making about this particular line. As its the two children of `<header>` that you're trying to "pad" and vertically align, it makes sense to add the padding directly to them, and not to *their* children.

In other words, there's no reason why you shouldn't move this `padding-top` you have here (on line 78) from each individual `<li>` to the parent, the `<nav>`.

Consider this CSS:

```
header {
  background-color: #033048;
}

h1 {
  float: left;
  padding: 12px 0 12px 70px;
  ...etc...
}

nav {
  float: right;
  padding: 28px 75px 28px 0;
}
```

You get a result like this:

![image](https://cloud.githubusercontent.com/assets/5670279/17484419/9c7eb37e-5d81-11e6-97ea-2b896f43a530.png)

... I've selected the `<nav>` in the Chrome Dev tools, so you can see the other benefit: by using padding, we can guarantee that the nav is perfectly vertically aligned (via equal amounts of padding `top` and `bottom`) as opposed to working out the maths of `(height of header - height of chosen font of nav) / 2`